lib/sanity.ts

```diff
@@ -0,0 +1,17 @@
+import { createClient } from "next-sanity";
+import imageUrlBuilder from "@sanity/image-url";
+
+const projectId = "aj1vwaqb";
+const dataset = "production";
+const apiVersion = "2023-01-01";
+
+export const client = createClient({
+  projectId,
+  dataset,
+  apiVersion,
+  useCdn: true,
+});
+
+const builder = imageUrlBuilder(client);
+
+export const urlFor = (source: any) => builder.image(source);
```

Comment:
Can you type this without using any?

Reply:
@Nassim-Saboundji yes! Just set it to string. Meant to change this before committing